streamDevice/src/StreamEpics.cc

```diff
@@ -944,12 +944,13 @@ getFieldAddress(const char* fieldname, StreamBuffer& address)
     else
     {
         // FIELD in this record or VAL in other record
-        char fullname[PVNAME_SZ + 1];
-        sprintf(fullname, "%s.%s", name(), fieldname);
+        size_t fullnameLen = PVNAME_SZ + 1;
```

Comment:
You are right, of course. I keep forgetting not every compiler got to 1999 yet :) Also, should `epicsSnprintf` be used instead?